### DISCUSSION TITLE
What kind of performance are you getting with what setup?

### DISCUSSION THREAD

**Discussion by JpTiger** [Raicuparta/nomai-vr#387] [Q&A]
Just out of curiosity how have you all be playing this and what kind of performance do you get out of your setup?

I've been using a first gen Oculus Quest, wirelessly streaming via [AMD ReVive VR](https://github.com/GPUOpen-LibrariesAndSDKs/Radeon-ReLive-VR) from my RX 480 and i5 7th gen. I played the game beginning to end with almost no issues, even on relatively high settings. Which is insane.

That said, I have a high tolerance for what other people might call issues. I don't tend to notice framerate particularly in most games unless it dips into the teens, and the Quest is capped at 72Hz anyway. I consider issues to be one where I literally cannot do a thing I'm supposed to, and that only happened once. Without getting too spoilery, it involved staying in the middle of something red to progress. I did have some weird stuttery behavior on the pause screen and menus, but totally manageable.

What have your experiences been like?

**Comment by Raicuparta** [comment]
It's possible that there will be an OWML update that improves performance significantly. I recently realized that there's some problem with OWML causing high CPU usage. Hopefully will help some people! That said, it will likely still be a very demanding game in VR.

EDIT: Added this "fix" (a hack really) to 1.0.5.
